test/integration/subscriptions.test.js

```diff
@@ -326,6 +327,69 @@ describe('Integration: subscriptions', () => {
         expect(subscription.isEnabledForGitHubEvent('issues')).toBe(true);
       });
 
+      test('successfully subscribing and unsubscribing with label', async () => {
+        const { Installation } = models;
+
+        installation = await Installation.create({
+          githubId: 1,
+          ownerId: fixtures.repo.owner.id,
+        });
+
+        nock('https://api.github.com').get('/repos/bkeepers/dotenv/installation').times(2).reply(200, {
+          id: installation.githubId,
+          account: fixtures.repo.owner,
+        });
+        nock('https://api.github.com').get('/repos/bkeepers/dotenv').times(2).reply(200, fixtures.repo);
+
+        await request.post('/slack/command').use(slackbot)
+          .send(fixtures.slack.command({
+            text: 'subscribe bkeepers/dotenv +label:"help wanted" +label:todo +label:priority:MUST +label:\'good first issue\'',
```

Comment:
Are we still allowing multiple label filters for now?

Reply:
For some weird reasons the tests are passing even though we do not allow that anymore.
I have to take a look, why the integration test was passing, despite my changes 🤔

Reply:
Fixed in https://github.com/integrations/slack/pull/911/commits/dc4fe0f17be107a46b63e0306caf5c18a3c3e9d9